aea/crypto/helpers.py

```diff
@@ -129,94 +73,21 @@ def _try_validate_cosmos_private_key_path(
             raise
 
 
-def _try_validate_private_key_path(
-    ledger_id: str, private_key_path: str, exit_on_error: bool = True
-) -> None:
-    """
-    Try alidate a private key path.
-
-    :param ledger_id: one of 'fetchai', 'ethereum'
-    :param private_key_path: the path to the private key.
-    :return: None
-    :raises: ValueError if the identifier is invalid.
-    """
-    if ledger_id == FETCHAI:
-        _try_validate_fet_private_key_path(private_key_path, exit_on_error)
-    elif ledger_id == ETHEREUM:
-        _try_validate_ethereum_private_key_path(private_key_path, exit_on_error)
-    elif ledger_id == COSMOS:
-        _try_validate_cosmos_private_key_path(private_key_path, exit_on_error)
-    else:
-        raise ValueError(
-            "Ledger id {} is not valid.".format(repr(ledger_id))
-        )  # pragma: no cover
-
-
-def _create_private_key(ledger_id: str, private_key_file: Optional[str] = None) -> None:
+def create_private_key(ledger_id: str, private_key_file: Optional[str] = None) -> None:
```

Comment:
now this is a public "helper" method

Reply:
Why do we not expose this on the crypto object instead?

Reply:
You mean for instance why we don't implement a class method in the `Crypto` interface, like `Crypto.load(ledger_id, private_key_file)`>

It is surely doable. However, I think that would break the single-responsibility principle for the `Crypto` interface. It would add a couping between the `Crypto` interface and the `aea.crypto.(register|make)` APIs. If we later change the `aea.crypto.registry`, we have to change `Crypto.load` as well.